.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
```

Comment:
non blocking: consider a ticket to merge with the other pipeline and conditionally publish on triggers and pasing tests.

Reply:
Its currently set to only allow a merge if the tests succeed, so I figure that's close enough for now.  Agreed that gating the real release on tests would be ideal.